e2e-tests-rust/tests/lib.rs

```diff
@@ -333,3 +334,15 @@ async fn cant_load_into_existing_state() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn set_chain_id() -> anyhow::Result<()> {
+    let provider = init_testing_provider(|node| node.block_time(3)).await?;
```

Comment:
Also, I suggest not using interval sealing in tests where it does not matter just so we don't have to wait 3 seconds unnecessarily:
```suggestion
    let provider = init_testing_provider(identity).await?;
```